Warehouse/WarehouseImplementation.cpp

```diff
@@ -428,9 +433,11 @@ namespace WPEFramework
             {
                 std::string msg = "file " CUSTOM_DATA_FILE " doesn't have any lines with paths";
                 LOGERR("%s", msg.c_str());
+                error = msg;
```

Comment:
The msg is only used for LOGERR(), instead use error in LOGERR() and remove msg.